src/services/errors/interfaces/base-error.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface BaseError {
+  message: string;
+  statusCode: number;
+  arguments: string[];
+}
```

Comment:
Is this based on `ErrorDetails` of the Safe Client Gateway? https://github.com/safe-global/safe-client-gateway/blob/d28bf1fb9feb069217ef52db1fde13bfb2b2aa58/src/utils/errors.rs#L30-L38

Reply:
Not explicitly, I was reviewing some errors and did the mapping. Do you know what is this `debug: Option<Value>` property? Should we map it as well?

Reply:
I think we can start with this payload but instead of `statusCode` let's just have `code` to match what is serialised to the clients. We can get back to this if we need to.

I would not add the `debug` for now 👍